console/network/Cargo.toml

```diff
@@ -34,18 +33,15 @@ version = "0.9.13"
 [dependencies.snarkvm-console-types]
 path = "../types"
 version = "0.9.13"
-default-features = false
 features = [ "field", "group", "scalar" ]
```

Comment:
These two above changes can be removed as the specifying flags automatically removes the `default` feature